src/app/learn/get-started/advanced/sticky-settings/overview/index.html

```diff
@@ -0,0 +1,84 @@
+<stache
+  pageTitle="Overview"
+  showTableofContents="true"
+  navOrder="10">
+
+  <stache-page-summary>
+    <p>
+      Sticky settings enable you to save user settings across sessions in flyouts, grids, and tile dashboards. Sticky settings preserve the <a stacheRouterLink="/learn/get-started/advanced/sticky-settings/integrations/skyux-components/flyouts">width of flyouts</a>, the <a stacheRouterLink="/learn/get-started/advanced/sticky-settings/integrations/skyux-components/grids">column order and the sorting of columns in grids</a>, and the <a stacheRouterLink="/learn/get-started/advanced/sticky-settings/integrations/skyux-components/tiles">layout and collapsed state of tiles in tile dashboards</a>.
+    </p>
+  </stache-page-summary>
+
+  <sky-alert>
+    For Blackbaud developers, the sticky settings functionality is available first-class through the UI Config Service that is for internal Blackbaud use only. Blackbaud developers can learn how to implement sticky settings on <a href="https://docs.blackbaud.com/ui-config-service-docs/">the UI Config Service website</a>.
+  </sky-alert>
+  <p>
+    To implement sticky settings, you must provide a class for <sky-code>SkyUIConfigService</sky-code> in your SPA's <sky-code>app-extras.module.ts</sky-code> file.
+  </p>
+  <sky-code-block languageType="typescript">
+    import {
+      AppUIConfigService
+    } from './shared/services/app-ui-config.service';
+
+    import {
+      SkyUIConfigService
+    } from '@skyux/core';
+
+    @NgModule({
+      providers: [
+        {
+          provide: SkyUIConfigService,
+          useClass: SkyAppUIConfigService
+        }
+      ]
+    })
+  </sky-code-block>
+  <p>
+    Next, you implement a service to retrieve stored settings. The following example provides <sky-code>AppUIConfigService</sky-code> to utilize the <a href="https://github.com/dscheerens/ngx-webstorage-service">ngx-webstorage-service</a>. You could just as easily implement the service to call your backend if you store settings across devices.
+  </p>
+  <sky-code-block languageType="typescript">
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  LOCAL_STORAGE,
+  StorageService
+} from 'ngx-webstorage-service';
+
+import {
+  SkyAppConfig
+} from '@skyux/config';
+
+@Injectable()
+export class SkyAppUIConfigService {
```

Comment:
Let's rename this class to 'AppUIConfigService`.

Reply:
Good catch. I removed "Sky."